hackernews/program.py

```diff
@@ -0,0 +1,45 @@
+import os
+import requests
+import time
+import urllib.parse
+
+from autokitteh.slack import slack_client
+
+
+slack = slack_client("slack_connection")
+SLACK_CHANNEL = os.getenv("SLACK_CHANNEL")
+
+
+def on_command(event):
+    topic = event.data.text  # Extract the topic from the Slack command
+    existing_articles = set()
+    fetch_articles(topic, existing_articles)
+
```

Comment:
I think it might be helpful for the user to receive a message at this point that says something like "request received; as soon as a new article on <topic> is posted, you will be notified."
Otherwise it's hard to know that something is happening.

Reply:
Maybe in the beginning after we trigger the function add request received so the user knows that it's working?